atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/ApiTestSchema.java

```diff
@@ -78,13 +78,13 @@ private static Schema generateSchema() {
                 rowName();
                 Stream.of(ValueType.values())
                         .filter(type -> type != ValueType.BLOB && type != ValueType.STRING)
-                        .forEachOrdered(type -> rowComponent("component" + type.ordinal(), type));
+                        .forEachOrdered(type -> rowComponent("component" + type.getIndex(), type));
```

Comment:
can't we get rid of the index here and just use something like `type.toString()` or is the ordinal part of our API?

Reply:
It seems like this is a test for the API but it is a internal test so we can make this change. We are not changing the actual API. Only changing how we are testing it.